src/index.js

```diff
@@ -68,14 +70,32 @@ function mnemonicToEntropy(mnemonic, wordlist) {
     if (!wordlist) {
         throw new Error(WORDLIST_REQUIRED);
     }
-    const words = normalize(mnemonic).split(' ');
+
+    var mnemonicAsBuffer = typeof mnemonic === 'string'
+    ? Buffer.from(unorm.nfkd(mnemonic), 'utf8')
```

Comment:
It looks like there are some linting issues here, the indentation is off. Also we should be using `const` over `var` where possible, since the rest of the package has since been updated to use `let`/const`